static/styles/popovers.css

```diff
@@ -472,14 +462,6 @@ ul {
         }
     }
 
-    &:not(.no-fields) {
-        #body {
-            padding-bottom: 10px;
-            padding-top: 4px;
-            margin-top: 10px;
-        }
```

Comment:
We can probably remove the `no-fields` class from the template and other CSS for that class as well, since there is no `hr` separator now for separating the custom profile fields and neither we reduce the height of modal if no custom fields are present. The code was added in [this commit](https://github.com/zulip/zulip/commit/c0edcf6fe4872b3e9cc3ac6330e377c5ed9b3d05)

Reply:
I agree it looks like that class is no longer useful; tested and confirmed, so I'll remove those before merging this.

Reply:
Thanks for the fix @timabbott!